.github/workflows/app-e2e.yml

```diff
@@ -79,8 +79,6 @@ jobs:
         uses: actions/cache@v3
         env:
           cache-name: cache-node-modules
-          # Workaround for bug https://github.com/typicode/husky/issues/991
-          HUSKY: 0
         with:
```

Comment:
Just to confirm, we don't need `HUSKY: 0` anymore and tests successfully work without it, do we?

Reply:
I researched the issue in git threads, and have found that this is not actual already. I tried both options (with it and without), for us - no difference.